modules/apps/commerce/commerce-theme-minium/commerce-theme-minium-impl/.eslintrc.js

```diff
@@ -13,5 +13,6 @@
  */
 
 module.exports = {
-	extends: ['@liferay/eslint-config/metal'],
+	extends: ['plugin:@liferay/metal'],
+	plugins: ['@liferay'],
 };
```

Comment:
What is this `@liferay` referencing here? Should it reference `@aui` too then?

Reply:
That is referencing `@liferay/eslint-plugin`.

Eslint will pickup `eslint-plugin-*` and `@*/eslint-plugin` packages where `*` is the name. There is no need to reference `@aui` here because the no-aui rules we are just not actually a separate plugin, we just enable them in a different config.